rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -41,14 +41,10 @@ trait Reduce[M[_]] {
 
 }
 
-class DebruijnInterpreter[M[_]](
+class DebruijnInterpreter[M[_]: Sync: Parallel: _cost](
```

Comment:
I'm just curious - why such unusual name for type class? `_cost`

Reply:
This is very good question.

We also have `_error`. Maybe because both are aliases.
https://github.com/rchain/rchain/blob/95ac99c4c991df111c3d7278827758a7e3607997/rholang/src/main/scala/coop/rchain/rholang/interpreter/package.scala#L7

I tried to replace `_error` with `Sync but it's not that easy, it seems that MonadState replaced with Ref in _cost can simplify it. I was playing with Ref already https://github.com/tgrospic/rchain/pull/12.